databases/week1/week1.sql

```diff
@@ -0,0 +1,61 @@
+1.Find out how many tasks are in the task table
```

Comment:
```suggestion
-- 1.Find out how many tasks are in the task table
```
This should be a mysql comment so, your .sql file can be still valid.